### PR TITLE
Add ordered delivery perf test

### PR DESCRIPTION
**Pull request by pierDipi**
I'd like to test the ordered delivery and how it compares to the unordered one.

### CODE REVIEW

File: testcases/kafka-broker-ordered/100-broker.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+apiVersion: eventing.knative.dev/v1
+kind: Broker
+metadata:
+  annotations:
+    # case-sensitive
+    eventing.knative.dev/broker.class: Kafka
+  name: my-kafka-broker
```
Comment (matzew) [inline testcases/kafka-broker-ordered/100-broker.yaml:7]: could we name that differently ? 

all brokers have "unique" names

File: testcases/kafka-broker-ordered/200-trigger.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+apiVersion: eventing.knative.dev/v1
+kind: Trigger
+metadata:
+  name: trigger-ordered-all-events-for-kafka-broker
+  annotations:
+    kafka.eventing.knative.dev/delivery.order: ordered
+spec:
+  broker: my-kafka-broker
```
Comment (matzew) [inline testcases/kafka-broker-ordered/200-trigger.yaml:8]: update here after rename too :smile:

File: testcases/kafka-broker-ordered/100-broker.yaml (hunk 1 above)
Comment (pierDipi) [reply testcases/kafka-broker-ordered/100-broker.yaml]: Thanks, done!

**Review by matzew — APPROVED** [review]
/lgtm
/approve

**Pull request merged**